addon/components/pie-chart.js

```diff
@@ -483,9 +483,17 @@ export default ChartComponent.extend(FloatingTooltipMixin,
     groups.select('path').attr(this.get('sliceAttrs'));
     
     var labelWidth = this.get('labelWidth');
+    var maxLabelWidth = this.get('outerWidth')/2 - this.get('labelPadding');
```

Comment:
Sure. Let me know if it makes more sense to turn this into a computed property. Happy to do so!